game.py

```diff
@@ -17,25 +17,56 @@ def snowman(snowman_word):
     """Complete the snowman function
     replace "pass" below with your own code
     It should print 'Congratulations, you win!'
-    If the player wins and, 
+    If the player wins and,
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_list = []
+    wrong_guesses_count = len(wrong_guesses_list)
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    victory = False
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES and not victory:
+        user_guess = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if user_guess not in snowman_word:
+            wrong_guesses_list.append(user_guess)
+            wrong_guesses_count = len(wrong_guesses_list)
+            print_snowman_graphic(wrong_guesses_count)
```

Comment:
Something that we could add here to enhance the user experience is adding a list of the wrong guesses each wrong guess. This would help the user not make multiple wrong guesses of the same letter.